.gitignore

```diff
@@ -1 +1,3 @@
 .idea/
+requirements.lock
```

Comment:
No need to have this. The commit that introduced this file should be removed entirely to get it out of history.